curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9bf80558d780e848b2987.md

```diff
@@ -22,13 +22,13 @@ assert.match(calculateCalories.toString(), /breakfastNumberInputs\s*=/);
 You should call `document.querySelectorAll()` with `#breakfast input[type=number]`.
 
 ```js
-assert.match(calculateCalories.toString(), /document\.querySelectorAll\(\s*('|")#breakfast input\[\s*type=number\s*\]\s*\1\s*\)/);
+assert.match(calculateCalories.toString(), /document\.querySelectorAll\(\s*('|")#breakfast input\[\s*('|")type=number\s*\]\s*\1\s*\)/);
```

Comment:
```suggestion
assert.match(calculateCalories.toString(), /document\.querySelectorAll\(\s*('|"|`)#breakfast input\[\s*type=('|")number\2\s*\]\s*\1\s*\)/);
```

you are adding the quote in the wrong place as it is `type="number"`, also it needs a capture group at the end of the string to match the second quote

Reply:
Thank you so much for the review! I will make the required changes right away!